app/src/main/res/layout/activity_main.xml

```diff
@@ -1,36 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
+
+<LinearLayout
```

Comment:
Using linear layouts seems to break rotation on my device. I get a 4th soft input button that I have to click to rotate the screen.

Reply:
I've reverted it back to a constraint layout.  Let me know if that improves the rotations on your device